src/doc/dev_manual/UpdatingPlugins.rst

```diff
@@ -45,7 +45,7 @@ Tag                Purpose                                Supported components
  
 
 *xml2cmake* also supports the *Conditional* addition of an include directory, definition, link library or source file.
-The condition is a CMake variable that describes something related to the build: e.g. an OS-specification (*WIN32, LINUX, MACOS*), the availability of a third-party library (*HAVE_CONDUIT*) or a build option that can be toggled on/off (*VISIT_OSPRAY*). 
+The condition is a CMake variable that describes something related to the build: e.g. an OS-specification (*WIN32, LINUX, MACOS*), the availability of a third-party library (*HAVE_CONDUIT*) or a build option that can be toggled on/off. 
```

Comment:
Do we want a new example here now that `(*VISIT_OSPRAY*)` was deleted?

Reply:
Good idea, I added `VISIT_PARALLEL` in its place.